presentation_fr.adoc

```diff
@@ -36,13 +36,16 @@ Johanna Vauchel::
 === Lectra
 image:images/lectra-4.0.png[width=800]
 
+=== Lectra
+image:images/lectra-numbers.png[width=800]
+
 === Opensource
 
 [%step]
 * Use Foss projects to build our solutions
 ** define a Foss policy to manage this usage
 * Use Innersource projects
-** incubator for futur Opensource projects
+** incubator for future Opensource projects
 * Contributor and creator of Opensource projects
 ** Partner in the https://hermine-foss.org/ project
 ** Kapoeira ...
```

Comment:
Maybe add this slide as a conclusion?

=== Main urls
* Member of https://naos-cluster.com/
* Partner in the https://hermine-foss.org/ project
* Github : https://github.com/lectra-tech/
* Dockerhub : https://hub.docker.com/u/lectratech